routes/contact/index.js

```diff
@@ -4,3 +4,7 @@ const router = express.Router();
 const get = require('./get');
 
 router.get('/contact', get);
+
+module.exports = (req, res) => { //requires a module export
```

Comment:
So we built a router on lines 2 and 6, but at this point, what you're exporting out is not that router.